src/archivist/index.js

```diff
@@ -140,7 +144,13 @@ export default class Archivist extends events.EventEmitter {
       return;
     }
 
-    return this.recordVersion(terms, extractOnly);
+    await this.recordVersion(terms, extractOnly);
+
+    terms.sourceDocuments.forEach(sourceDocument => {
```

Comment:
> Why can't we just do `terms.sourceDocuments = []` or `terms.sourceDocuments = null`?

Reply:
Because doing so would result in losing information needed for the next run, such as `location`, `contentSelectors`, …